lapce-ui/src/explorer.rs

```diff
@@ -690,7 +692,50 @@ impl Widget<LapceTabData> for FileExplorerFileList {
                 let file_explorer = Arc::make_mut(&mut data.file_explorer);
                 let index = ((mouse_event.pos.y + self.line_height)
                     / self.line_height) as usize;
+
                 if mouse_event.button.is_left() {
+                    // Check double click delay
+                    let f = || -> bool {
```

Comment:
mouse_event has `count` which can be used to check double click or trigger click.

Reply:
Awesome. This should be a LOT simpler now.